packages/babel-core/test/config-chain.js

```diff
@@ -1024,24 +1020,64 @@ describe("buildConfigChain", function() {
         },
       );
 
+      test("should not load babel.config.mjs synchronously", async () => {
+        const { cwd, tmp, config } = await getTemp(
+          "babel-test-load-config-sync-babel.config.mjs",
+        );
+        const filename = tmp("src.js");
+
+        await config("babel.config.mjs");
+
+        expect(() => loadOptions({ filename, cwd })).toThrow(
+          /is only supported when running Babel asynchronously/,
```

Comment:
You can use `toThrowErrorMatchingSnapshot()` here and don't have to write the same string again.

Reply:
I can't use it because the error contains the path of the file in the tmp folder, which might change.